src/openforms/js/components/admin/form_design/variables/constants.js

```diff
@@ -85,6 +85,12 @@ const VARIABLE_SOURCES = {
   userDefined: 'user_defined',
 };
 
+const VARIABLE_SOURCES_GROUP_LABELS = {
+  userDefined: 'user variables',
+  component: 'component variables',
+  static: 'static variables',
+};
```

Comment:
this labels need to be translatable and translations extracted using react-intl (we have a script in `bin` for that)